### PR TITLE
SU-519: Implement penalty start date

### PR DESCRIPTION
**Pull request by hagafrank**
SU-519: Implement penalty start date

### CODE REVIEW

File: fineract-provider/src/main/java/org/apache/fineract/infrastructure/configuration/domain/ConfigurationDomainServiceJpa.java [hunk 1]
```diff
@@ -608,4 +608,11 @@ public Integer retriveIvaConfiguration() {
         return value;
     }
 
+    @Override
+    public LocalDate retrievePenaltyStartDate() {
+        final String propertyName = "penalty-start-date";
+        final GlobalConfigurationPropertyData property = getGlobalConfigurationPropertyData(propertyName);
```
Comment (oluexpert99) [inline fineract-provider/src/main/java/org/apache/fineract/infrastructure/configuration/domain/ConfigurationDomainServiceJpa.java:614]: check if config is enabled @hagafrank

Comment (oluexpert99) [reply fineract-provider/src/main/java/org/apache/fineract/infrastructure/configuration/domain/ConfigurationDomainServiceJpa.java:614]: if config is disabled  , we can return a very early date

Comment (hagafrank) [reply fineract-provider/src/main/java/org/apache/fineract/infrastructure/configuration/domain/ConfigurationDomainServiceJpa.java:614]: Thanks @oluexpert99, I have added code to handle case where the config is disabled

**Review by oluexpert99 — APPROVED** [review]
LGTM

**Pull request merged**